.github/workflows/publish-installers.yml

```
@@ -0,0 +1,281 @@
+          curl -fsSL https://cli.github.com/packages/githubcli-archive-keyring.gpg | sudo dd of=/usr/share/keyrings/githubcli-archive-keyring.gpg
+          sudo chmod go+r /usr/share/keyrings/githubcli-archive-keyring.gpg
+          echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/githubcli-archive-keyring.gpg] https://cli.github.com/packages stable main" | sudo tee /etc/apt/sources.list.d/github-cli.list > /dev/null
+          sudo apt-get update -y
+          sudo apt-get install -y gh
+
+      - name: Save download and publish script (fixed heredoc)
+        run: |
+          mkdir -p .github/scripts
+          cat > .github/scripts/download_and_publish_verbose.sh <<'EOF'
+#!/usr/bin/env bash
+set -euo pipefail
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)